client/post-editor/copy-post/accordion.jsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import cloneDeep from 'lodash/cloneDeep';
+import isEmpty from 'lodash/isEmpty';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import PostActions from 'lib/posts/actions';
+import PostSelector from 'my-sites/post-selector';
+
+module.exports = React.createClass( {
```

Comment:
- Should be consistent with your module import/export syntax. Specifically, I'd recommend using `export default` instead of `module.exports`
- I've been discouraging folks from using `React.createClass`, as [looking through the React core meeting notes](https://github.com/reactjs/core-notes/search?utf8=%E2%9C%93&q=createclass) it becomes very obvious that it's a discouraged feature. Consider instead the [`React.Component` class](https://facebook.github.io/react/docs/reusable-components.html#es6-classes) or, if your component doesn't need lifecycle or state, a [pure stateless function component](https://facebook.github.io/react/docs/reusable-components.html#stateless-functions)